lib/pdf_cover.rb

```diff
@@ -72,6 +72,7 @@ def included(base)
       if carrierwave_defined?(base)
         base.extend ClassMethods::CarrierWave
       elsif paperclip_defined?
+        require "paperclip/pdf_cover_processor"
```

Comment:
Otherwise the Processor is never loaded. The only reason the tests were passing was because we manually required it in the tests.